sorbet/rbi/gems/actioncable@5.2.4.3.rbi

```diff
@@ -1,6 +1,6 @@
 # DO NOT EDIT MANUALLY
 # This is an autogenerated file for types exported from the `actioncable` gem.
-# Please instead update this file by running `tapioca generate`.
+# Please instead update this file by running `tapioca sync`.
```

Comment:
I though the right command was `sync` and not `generate`? I can revert those changes if it should be `generate`.